background.ts

```diff
@@ -147,12 +148,149 @@ Electron.app.whenReady().then(async() => {
     }
     callback(result);
   });
+  buildApplicationMenu();
   window.openPreferences();
 
   setupKim(k8smanager);
   setupUpdate(cfg);
 });
 
+function buildApplicationMenu(): void {
+  const menuItems: Array<MenuItem> = [];
+  const isMac = process.platform === 'darwin';
+
+  if (isMac) {
+    menuItems.push(new MenuItem({
+      label:   Electron.app.name,
+      submenu: [
+        {
+          role:  'about',
+          label: `About ${ Electron.app.name }`,
+        },
+        { type: 'separator' },
+        { role: 'services' },
+        { type: 'separator' },
+        { role: 'hide' },
+        { role: 'hideOthers' },
+        { role: 'unhide' },
+        { type: 'separator' },
+        { role: 'quit' },
+      ],
+    }));
+  }
+  menuItems.push(new MenuItem({
+    label:   '&File',
+    submenu: [
+      { role: isMac ? 'close' : 'quit' },
+    ]
+  }));
+  if (isMac) {
+    menuItems.push(new MenuItem({
+      label:   'Edit',
+      submenu: [
+        { role: 'undo' },
+        { role: 'redo' },
+        { type: 'separator' },
+        { role: 'cut' },
+        { role: 'copy' },
+        { role: 'paste' },
+        {
+          role:    'pasteAndMatchStyle',
```

Comment:
I thought it would be more appropriate to show the standard menu items disabled, rather than not show them.

OTOH, showing a disabled menu item implies that under some circumstances said menu item will be enabled, which isn't going to happen.